File: infrastructure/src/main/java/com/github/dkurata38/liblio/infrastructure/domain/user/UserRecordMapper.java
package com.github.dkurata38.liblio.infrastructure.domain.user;

import org.apache.ibatis.annotations.Mapper;
import org.apache.ibatis.annotations.Param;

@Mapper
public interface UserRecordMapper {
	UserRecord selectByUsername(@Param("username") String username);
}

File: infrastructure/src/main/kotlin/com/github/dkurata38/liblio/infrastructure/user/UserRecordMapper.kt
package com.github.dkurata38.liblio.infrastructure.user

import org.apache.ibatis.annotations.Mapper
import org.apache.ibatis.annotations.Param

@Mapper
interface UserRecordMapper {
    fun selectByUsername(@Param("username") username: String): UserRecord?

    fun insert(userRecord: UserRecord): Int;
}
